### DISCUSSION TITLE
Does Koobiq/angular-components play well with TailwindCSS?

### DISCUSSION THREAD

**Discussion by touhidrahman** [koobiq/angular-components#351] [Q&A] [answered]
I just found out the library and it looks awesome! I would like to use it in my projects, however for better coding ergonomics I (& my team) use TailwindCSS heavily. So, does the library works well with tailwind? I can always find out by setting up a demo project, but just wanted to know the maintainers' position and planning on this.

Keep up the good work!

**Comment by pimenovoleg** [comment, marked as answer]
Thank you for your feedback, we really appreciate it! :)

Yes, we are planning to support TailwindCSS through a dedicated plugin (plugin itself might not be ready this year). Currently, in version 18, we are actively implementing the use of CSS variables, which should help with configuring TailwindCSS (we’ll prepare an example of how to use it).

Keep up the great work with your projects!
